jsk_fetch_robot/jsk_fetch_startup/launch/fetch.launch

```diff
@@ -111,7 +111,12 @@
                      false, false, false]
       odom0_nodelay: true
       odom0_differential: true
+      permit_corrected_publication: false
```

Comment:
@sktometometo 
Setting this param to `false` suppresses the warning. Was your first commit wrong?